### ISSUE TITLE
NVDA does not announce title for modal dialog in web when pressed insert+T 

### ISSUE THREAD

**Issue by Rasubra8** [nvaccess/nvda#7135]
For modal dialogs, NVDA does not announce the title while
pressing insert+T, in IE, FF, and Chrome as well.
In web pages, NVDA is announcing the page title instead of the dialog title.
Where as, in desktop, it announces the dialog title. Not the application title.
Generally, the expected behavior is the desktop behavior should be
replicated in the web as well. Right?

**Comment by Rasubra8** [comment]
Any update on this issue please?

**Comment by ehollig** [comment]
CC @jcsteh I know some work has been done on modal dialogs. Any updates?

**Comment by jcsteh** [comment]
This is unrelated to the current work on web dialogs.

The expected behaviour for the report title (NVDA+t) command is that it reports the title of the current application or foreground window. On the web, the foreground window is always the browser itself except when an operating system dialog (e.g. Save As) is displayed. In terms of user experience, I think there's a good argument for making this also report the active document/dialog on the web, but we'd consider this a feature request, not a bug. Labelling appropriately.
